examples/3d/pbr.rs

```diff
@@ -69,14 +72,115 @@ fn setup(
         },
         ..default()
     });
-    // camera
-    commands.spawn(Camera3dBundle {
-        transform: Transform::from_xyz(0.0, 0.0, 8.0).looking_at(Vec3::default(), Vec3::Y),
-        projection: OrthographicProjection {
-            scale: 0.01,
+
+    // labels
+    commands.spawn(
+        TextBundle::from_section(
+            "Perceptual Roughness",
+            TextStyle {
+                font: asset_server.load("fonts/FiraMono-Medium.ttf"),
+                font_size: 36.0,
+                color: Color::WHITE,
+            },
+        )
+        .with_style(Style {
+            position_type: PositionType::Absolute,
+            position: UiRect {
+                top: Val::Px(20.0),
+                left: Val::Px(100.0),
+                ..default()
+            },
             ..default()
-        }
-        .into(),
+        }),
+    );
+
+    commands.spawn(TextBundle {
+        text: Text::from_section(
+            "Metallic",
+            TextStyle {
+                font: asset_server.load("fonts/FiraMono-Medium.ttf"),
+                font_size: 36.0,
+                color: Color::WHITE,
+            },
+        ),
+        style: Style {
+            position_type: PositionType::Absolute,
+            position: UiRect {
+                top: Val::Px(130.0),
+                right: Val::Px(0.0),
+                ..default()
+            },
+            ..default()
+        },
+        transform: Transform {
+            rotation: Quat::from_rotation_z(std::f32::consts::PI / 2.0),
+            ..default()
+        },
         ..default()
     });
+
+    commands.spawn((
+        TextBundle::from_section(
+            "Loading Environment Map...",
+            TextStyle {
+                font: asset_server.load("fonts/FiraMono-Medium.ttf"),
+                font_size: 36.0,
+                color: Color::RED,
+            },
+        )
+        .with_style(Style {
+            position_type: PositionType::Absolute,
+            position: UiRect {
+                bottom: Val::Px(20.0),
+                right: Val::Px(20.0),
+                ..default()
+            },
+            ..default()
+        }),
+        EnvironmentMapLabel,
+    ));
+
+    // camera
+    let environment_map = EnvironmentMap {
+        diffuse_map: asset_server.load("environment_maps/pisa_diffuse.ktx2"),
```

Comment:
Since this example requires the `ktx2` feature, I wonder if it's possible to tell cargo to use it automatically for this example

Reply:
To be clear, I know it's a required feature, but cargo still asked me to enter it manually

Reply:
Not possible sadly https://github.com/rust-lang/cargo/issues/4663

Reply:
😢